xcube/webapi/ows/stac/controllers.py

```diff
@@ -532,6 +588,29 @@ def _get_single_dataset_collection(
                 'title': f'Coverage for the dataset "{dataset_id}" using '
                 f'OGC API – Coverages, as GeoTIFF',
             },
+            {
+                'rel': 'http://www.opengis.net/def/rel/ogc/1.0/schema',
+                'href': f'{base_url}{PATH_PREFIX}/collections/'
+                        f'{dataset_id}/schema?f=json',
+                'type': 'application/json',
+                'title': 'Schema (as JSON)',
+            },
+            {
+                'rel': 'http://www.opengis.net/def/rel/ogc/1.0/'
+                       'coverage-rangetype',
+                'href': f'{base_url}{PATH_PREFIX}/collections/'
+                        f'{dataset_id}/coverage/rangetype?f=json',
+                'type': 'application/json',
+                'title': 'Range type of the coverage',
+            },
+            {
+                'rel': 'http://www.opengis.net/def/rel/ogc/1.0/'
+                       'coverage-domainset',
+                'href': f'{base_url}{PATH_PREFIX}/collections/'
+                        f'{dataset_id}/coverage/domainset?f=json',
+                'type': 'application/json',
+                'title': 'Domain set of the coverage',
```

Comment:
Same here, maybe move the URL strings into constants?

Reply:
Done.